boards/arm/efr32xg24_dk2601b/efr32xg24_dk2601b_defconfig

```diff
@@ -17,3 +17,6 @@ CONFIG_CMU_HFCLK_LFXO=y
 CONFIG_HW_STACK_PROTECTION=y
 CONFIG_PINCTRL=y
 CONFIG_REGULATOR=y
+CONFIG_SI7006=y
+CONFIG_SI7210=y
+CONFIG_VEML7700=y
```

Comment:
These `CONFIG` are automatically selected if each sensor has `status = "okay"`. There is no need to add this.
See https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/sensor/veml7700/Kconfig#L9 for reference.

Reply:
Got it, thank you